Eigen does not have cmake macro
Add link directory for assimp

--- CMakeLists.txt~	2018-05-14 11:48:00.000000000 +0200
+++ CMakeLists.txt	2018-09-20 11:37:24.430345718 +0200
@@ -21,7 +21,8 @@
 
 find_package(console_bridge REQUIRED)
 
-find_package(Eigen3 REQUIRED)
+find_package(PkgConfig)
+pkg_search_module(EIGEN3 REQUIRED eigen3)
 
 find_package(octomap REQUIRED)
 
@@ -54,6 +55,7 @@
 include_directories(include)
 include_directories(${EIGEN3_INCLUDE_DIR} ${Boost_INCLUDE_DIR} ${ASSIMP_INCLUDE_DIRS} ${OCTOMAP_INCLUDE_DIRS})
 include_directories(${catkin_INCLUDE_DIRS} ${console_bridge_INCLUDE_DIRS})
+link_directories(${ASSIMP_LIBRARY_DIRS})
 
 add_library(${PROJECT_NAME}
   src/bodies.cpp
